### DISCUSSION TITLE
Permission is denied

### DISCUSSION THREAD

**Discussion by n-3-15-0-n** [rizwansoaib/whatsapp-monitor#94] [Q&A]
I got a pop-up with "Permission is denied" when the target is online. May I know how to fix it?

**Comment by rizwansoaib** [comment]
Please allow permission click on website url left side tab icon then
settings permission. From web.whatsapp.com

On Thu 15 Jul, 2021, 11:09 AM n-3-15-0-n, ***@***.***> wrote:

> I got a pop-up with "Permission is denied" when the target is online. May
> I know how to fix it?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/94>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZGOKDRHLYQQ4Y5ND6DTXZYARANCNFSM5AMZ2K5A>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.

**Reply by n-3-15-0-n** [reply]
Thank you for your reply. It works great
